ui/src/datasets.tsx

```diff
@@ -133,6 +133,10 @@ export function Datasets() {
   });
 
   const allAttributesChecked = () => {
+    if (conceptSetOccurrences.length === 0) {
```

Comment:
behaviour change?

Reply:
This just fixes the default look of the checkbox when there are no concept sets selected vs. when all of the values of a concept set are unselected.